### DISCUSSION TITLE
Who is using `iti`? Share your use case 🙏

### DISCUSSION THREAD

**Discussion by molszanski** [molszanski/iti#35] [Show and tell]
Let me start. 

I use it at multiple projects at packhelp.com. 

It powers our online 3D packaging editor and multiple e-commerce projects. 

We mainly use it for fronted, but there is one backend microservice that is using it now

**Comment by danielmahon** [comment]
I just refactored a couple dozen Firebase functions inside an NX workspace that were a mix of NestJS and no IoC to `iti` and I literally "sighed with relief" when I saw the results. It was so clean, so simple, type-safe, and so easy to follow. Definitely going to keep exploring `iti` and other possible use-cases.

**Reply by molszanski** [reply]
Happy to hear that and thank you for sharing!!! I have a similar experience when I refactor tightly coupled projects to it :)
